gapic/src/main/com/google/gapid/perfetto/models/CounterInfo.java

```diff
@@ -38,13 +39,29 @@
   public final double max;
   public final double avg;
 
+  private String GenerateCounterName(String name) {
```

Comment:
This is very specific to your counters. Let's not do it this way, but handle it like we are handling the system memory and battery counters. If, for now, you just want to show them as counters, simply expand the `CounterPanel` to take a name override and in `Tracks` look for counters with the specific names.

For one thing, it's impossible to tell from this what the actual labels will be.

Reply:
Done.